### ISSUE TITLE
[Support] Expected format of `/sys/devices/virtual/dmi/id/product_name`

### ISSUE THREAD

**Issue by samveen** [xcat2/xcat-core#3453]
Due to a comment on #3449 , I took a look at the my site's `VPD` and found that `vpd.mtm` is empty for ALL nodes. Taking a look at `xCAT-genesis-scripts/bin/dodiscovery`, I found that it expects the `/sys/devices/virtual/dmi/id/product_name` file to contain `[` and `]`. Can you give me an example of what the expected format of the contents of this file are?

**Comment by immarvin** [comment]
hi @zet809 , would you please answer the question? thx

**Comment by zet809** [comment]
HI, @samveen , the file looks like this:
```
# cat /sys/devices/virtual/dmi/id/product_name
 -[786310X]-
```

**Comment by samveen** [comment]
@immarvin @zet809 Thank you for the quick response.

As luck would have it, the issue I face are fixed already by PRs #2479 and #2712  ( included in `xCAT-2.13.3`, while we use `xCAT-2.13.2`)

Closing this.
